pkg/cas/resolver/resolver.go

```diff
@@ -24,13 +25,23 @@ var logger = log.New("cas-resolver")
 // Resolver represents a resolver that can resolve data in a CAS based on a CID and WebCAS URL.
 type Resolver struct {
 	localCAS   casapi.Client
+	ipfsReader ipfsReader
 	httpClient *http.Client
 }
 
+type ipfsReader interface {
+	Read(address string) ([]byte, error)
+}
+
+// TODO: Fix with webfinger
```

Comment:
Can you add (#364) to this TODO to help me remember to change this?

Reply:
Done